htmd/parameterization/cli.py

```diff
@@ -519,12 +519,12 @@ def _printEnergies(molecule, parameters, filename):
 Electro  : {ELEC_ENERGY:12.6g} kcal/mol
 VdW      : {VDW_ENERGY:12.6g} kcal/mol
 
-'''.format(BOND_ENERGY=energies['bond'],
-           ANGLE_ENERGY=energies['angle'],
-           DIHEDRAL_ENERGY=energies['dihedral'],
-           IMPROPER_ENERGY=energies['improper'],
-           ELEC_ENERGY=energies['elec'],
-           VDW_ENERGY=energies['vdw'])
+'''.format(BOND_ENERGY=np.round(energies['bond'], decimals=5),
```

Comment:
It looks inconsistent, you are rounding to 5 decimal places, but the formatting is set to `12.6g`. Wouldn't setting it to `12.5f`, the rounding will be done implicitly.

Reply:
No, that would cause issues with large numbers not using scientific notation. I played around with it a bit

Reply:
Unless you don't care about such large energies. In which case sure I can switch from `g` to `f` formatting